pkg/uroot/builder/gbb_test.go

```diff
@@ -9,11 +9,13 @@ import (
 
 	gbbgolang "github.com/u-root/gobusybox/src/pkg/golang"
 	"github.com/u-root/u-root/pkg/golang"
+	"github.com/u-root/u-root/pkg/testutil"
 	"github.com/u-root/u-root/pkg/ulog/ulogtest"
 	"github.com/u-root/u-root/pkg/uroot/initramfs"
 )
 
 func TestGBBBuild(t *testing.T) {
+	testutil.SkipIfNotRoot(t) // Why is this needed on CircleCI?
```

Comment:
Yo this just skips the test altogether in all environments, doesn't it? Don't do this?

Reply:
The unit test runs as not-root, so it's skipped. And when run in a VM, there's no Go compiler, so it's also skipped.

Reply:
But in gotest_test.go those tests are invoked again? And this is also a mechanism used to get coverage from some VM tests no? Also I wouldn't have to use it if you can tell me why it only specifically fails in CircleCI and nowhere else. I would rather do a `testutil.SkipIfInCircleCI(t)` if it existed.

Reply:
Update: GO111MODULE=off broke it as those additional files were not correctly recognized by gbb and therefore not copied over to the temporary source tree. Omitting that variable and using Golangs default (e.g. on) has no issues with it.